src/main/java/flaxbeard/steamcraft/client/render/RenderCanister.java

```diff
@@ -35,10 +35,10 @@ public void doRender(Entity var1, double x, double y, double z, float var8, floa
         item.hoverStart = 0.0F;
 
         GL11.glTranslated(0, 0.85F - (4F / 16F), 0);
-        if (this.renderManager.options.fancyGraphics == false) {
+        if (!this.renderManager.options.fancyGraphics) {
             GL11.glScalef(1.25F, 1.25F, 1.25F);
         }
-        if (this.renderManager.options.fancyGraphics == true || item.getEntityItem().getItem() instanceof ItemBlock) {
```

Comment:
thank you

(Santa, if you were the one who put "== true" in I'm gonna yell at you)